examples/LineChartSecondaryAxis/main.swift

```diff
@@ -16,8 +16,8 @@ var svg_renderer: SVGRenderer = SVGRenderer()
 var plotTitle: PlotTitle = PlotTitle()
 
 var lineGraph: LineGraph = LineGraph()
-lineGraph.addSeries(x1, y1, label: "Plot 1", color: Color.lightBlue, axisType: Axis.PRIMARY_AXIS)
-lineGraph.addSeries(x, y, label: "Plot 2", color: Color.orange, axisType: Axis.SECONDARY_AXIS)
+lineGraph.addSeries(x1, y1, label: "Plot 1", color: Color.lightBlue, axisType: Axis.Location.primaryAxis)
+lineGraph.addSeries(x, y, label: "Plot 2", color: Color.orange, axisType: Axis.Location.secondaryAxis)
 plotTitle.title = "SECONDARY AXIS"
```

Comment:
As an additional simplification, because Swift can infer types, I often leave off the type prefix for enum values. `.primaryAxis` instead of `Axis.Location.primaryAxis` or `.lightBlue` instead of `Color.lightBlue`. If type can't be inferred, or if specifying the type prefix makes things clearer, then I'll do it but otherwise I tend to drop that to focus on the value.

Reply:
Done. I also made the stackPatter in subplots an enum.